include/deal.II/grid/grid_tools.h

```diff
@@ -576,6 +581,28 @@ namespace GridTools
   transform(const Transformation &        transformation,
             Triangulation<dim, spacedim> &triangulation);
 
+#ifdef DEAL_II_WITH_CGAL
+  /**
+   * Create a deal.II Triangulation<3> out of a deal.II Triangulation<2,3>
+   * by filling it with tetrahedrons. Optional variable arguments `cgal_args`
+   * can be used to control size, quality, and distribution of the cells of
+   * the final triangulation.
```

Comment:
Can you copy here the blob about the parameters of mesh_criteria_3 that we have in the other function?